pkg/volume/nfs/nfs.go

```diff
@@ -164,6 +165,33 @@ func (nfsVolume *nfs) GetPath() string {
 	return nfsVolume.plugin.host.GetPodVolumeDir(nfsVolume.pod.UID, strings.EscapeQualifiedNameForDisk(name), nfsVolume.volName)
 }
 
+// Checks prior to mount operations to verify that the required components (binaries, etc.)
+// to mount the volume are available on the underlying node.
+// If not, it returns an error
+func (nfsMounter *nfsMounter) CanMount() error {
+	exe := exec.New()
+	missingBinaryMsg := "Binary required for mounting, does not exist on the node machine. Verify that your node machine has the required binaries before attempting to mount this volume type"
```

Comment:
I would remove the `missingBinaryMsg` here. Since it will otherwise be duplicated in each plugin, and have the plugins only return exactly what is missing. e.g. `/sbin/mount_nfs is missing`